tests/php/src/Internal/DependencyManagement/ExampleClasses/ClassWithConstructorArgumentWithoutTypeHint.php

```diff
@@ -13,10 +13,10 @@
 class ClassWithConstructorArgumentWithoutTypeHint {
```

Comment:
For consistence we should rename this class to something like `ClassWithSetDependenciesArgumentWithoutTypeHint`

Reply:
Thanks, I meant to rename these classes and tests accordingly but it slipped my mind by the time it came to actually do it 🤣